contrib/platform/test/com/sun/jna/platform/win32/COM/WbemcliTest.java

```diff
@@ -282,4 +291,22 @@ public void testWmiOperatingSystem() {
         assertEquals(Variant.VT_BOOL, os.getVtType(OperatingSystemProperty.PRIMARY));
         assertNotNull(os.getValue(OperatingSystemProperty.PRIMARY, 0));
     }
+
+    enum Win32_DiskDrive_Values {
+        Caption,
```

Comment:
Some coding convention analysis tools will complain that these (constant) `enum` values aren't all uppercase. I think it also may make the later test assertions more readable.  It still works as WMI queries are case-insensitive.

Reply:
I tend to disagree - the casing matches the version returned by `GetNames` - anyway adjusted.

Reply:
The earlier enums in the test class (that I wrote) are in all caps. :)  But I would agree with you that JNA tends to try to keep consistent with Windows cases so I'm ok with this.